wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -633,9 +624,8 @@ def bulk_action_choices(context, app_label, model_name):
     if bulk_action_more_list:
         more_button = ButtonWithDropdown(
             label=_("More"),
-            attrs={"title": _("View more bulk actions")},
-            classes={"bulk-actions-more", "dropup"},
-            button_classes={"button", "button-small"},
+            attrs={"title": _("More bulk actions")},
```

Comment:
Completely unrelated to other changes – using an `aria-label` starting with something else than the visual label would be very confusing for assistive technology users.